docs/how-to/drbd-host-networking.md

```diff
@@ -0,0 +1,100 @@
+# How to Use the Host Network for DRBD Replication
+
+This guide shows you how to use the host network for DRBD® replication.
+
+By default, DRBD will use the container network to replicate volume data, which causes a temporary disruption of
+replication when the LINSTOR® Satellite Pod is restarted. Your container network may also be too slow for replication.
+In these cases, you might want to configure DRBD replication to use host network.
```

Comment:
Also explain the advantages of container network replication so that the user can make an informed decision.